enterprise/server/raft/store/store.go

```diff
@@ -954,8 +954,29 @@ func (s *Store) syncRequestDeleteReplica(ctx context.Context, rangeID, replicaID
 		}
 		return err
 	})
-	if err != nil {
-		return status.InternalErrorf("failed to request delete replica for c%dn%d: %s", rangeID, replicaID, err)
+	return err
+}
+
+func (s *Store) syncRequestStopAndDeleteReplica(ctx context.Context, rangeID, replicaID uint64) error {
```

Comment:
nit: add a comment on what's the difference between this method and the syncRequestDelete method.